js/ui/slider/ui.slider.js

```diff
@@ -380,16 +381,34 @@ const Slider = TrackBar.inherit({
 
     _renderStartHandler: function() {
         const pointerDownEventName = addNamespace(pointerEvents.down, this.NAME);
+        const pointerUpEventName = addNamespace(pointerEvents.up, this.NAME);
         const clickEventName = addNamespace(clickName, this.NAME);
         const startAction = this._createAction(this._startHandler.bind(this));
         const $element = this.$element();
 
+        const processOnMovingComplete = () => {
+            if(this.option('callValueChange') === 'onMovingComplete') {
+                if(this._getActualValue()) {
+                    this.option('value', this._getActualValue());
+                    this._actualValue = undefined;
+                }
+            }
+        };
+
         eventsEngine.off($element, pointerDownEventName);
         eventsEngine.on($element, pointerDownEventName, e => {
             if(isMouseEvent(e)) {
                 startAction({ event: e });
             }
         });
+
+        eventsEngine.off($element, pointerUpEventName);
+        eventsEngine.on($element, pointerUpEventName, e => {
+            if(isMouseEvent(e)) {
```

Comment:
why is this condition necessary? Seems like it will not work on mobile devices, because pointerup is `touch` event on mobiles.

Reply:
Checked on iphone - works ok